apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -100,8 +101,37 @@ export default class CoreLibrary {
     width = Math.max(width, 50);
     const height = lines.length * textSize + padding * 2;
 
+    let triangleSize = 10;
+    let tipX = x;
+
+    // For the calculations below, keep in mind that x and y are located at the horizontal center and the top of the sprite, respectively.
+    // In other words, x and y indicate the default position of the bubble's triangular tip.
+    if (y > 400) {
+      y = 400;
+    }
```

Comment:
a couple of nits here:
- throughout this function, we should use the `APP_WIDTH` and `APP_HEIGHT` constants instead of hardcoding 400. those constants are in [p5lab/constants.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/constants.js#L10-L14)
- we can simplify these lines to `y = Math.min(y, 400)`